intelmq/tests/bots/experts/idea/test_expert.py

```diff
@@ -82,10 +83,10 @@ def test_conversion(self):
         # The ID in the generated Idea event is random, so we have to extract
         # the data from the "output" field and compare after removing ID's
         event = self.get_output_queue()[0]
-        self.assertIsInstance(event, str)
-        event_dict = json.loads(event)
+        self.assertIsInstance(event, bytes)
+        event_dict = msgpack.loads(event)
```

Comment:
Could we use `MessageFactory.unserialize` here? Then we are flexible with the default packer and we don't need `b"output"` below.